torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -638,6 +714,52 @@ bool isNodeParsible(const Node* node) {
   return IrParser::canParseNode(node);
 }
 
+// TODO: we should incorporate this to our parser as well;
```

Comment:
It's not clear what our parser refers to in this context: replace `our parser` to a more explicit pointer ?